pkg/cli-runtime/parsers/object.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parsers
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func JsonYamlObject(kv string) (interface{}, error) {
```

Comment:
I would make the function name `JsonYamlToObject` , its clear for caller about from and to types.

Reply:
Name updated